cloud: update product description and add a diagram (#9923)
pingcap/docs: tidb-cloud/tidb-cloud-intro.md

# TiDB Cloud Introduction

[TiDB Cloud](https://en.pingcap.com/tidb-cloud/) is a fully-managed Database-as-a-Service (DBaaS) that brings [TiDB](https://docs.pingcap.com/tidb/stable/overview), an open-source Hybrid Transactional and Analytical Processing (HTAP) database, to your cloud. TiDB Cloud offers an easy way to deploy and manage databases to let you focus on your applications, not the complexities of the databases. You can create TiDB Cloud clusters to quickly build mission-critical applications on Google Cloud Platform (GCP) and Amazon Web Services (AWS).

## Why TiDB Cloud

> TiDB Cloud allows users with little or no training to handle complex tasks such as infrastructure management and cluster deployment easily.

Reviewer: ```suggestion
TiDB Cloud allows you with little or no training to handle complex tasks such as infrastructure management and cluster deployment easily.
```

> - Developers and database administrators (DBAs) can rapidly analyze large volume of data across multiple datasets.

Reviewer: ```suggestion
- Developers and database administrators (DBAs) can rapidly analyze a large volume of data across multiple datasets.
```

> - Enterprises of all sizes can easily deploy and manage TiDB Cloud to adapt to your business growth all without prepayment.

Reviewer 2: Seems not necessary?
```suggestion
- Enterprises of all sizes can easily deploy and manage TiDB Cloud to adapt to your business growth without prepayment.
```

With TiDB Cloud, you can get the following key features:

- **Fully-Managed TiDB Service**

    Deploy, scale, monitor, and manage TiDB clusters with a few clicks, through an easy-to-use web-based management platform.

- **Real-Time Analytics**

>     Get real-time analytical query results with our Hybrid Transactional and Analytical Processing (HTAP) capabilities.

Reviewer 2: ```suggestion
    Get real-time analytical query results with Hybrid Transactional and Analytical Processing (HTAP) capabilities.
```

- **Fast and Customized Scaling**

>     Elastically and transparently scale to hundreds of nodes for critical workloads without changing business logic. And you can scale your performance and storage nodes separately to custom tailor to your business needs.

Reviewer 2: ```suggestion
    Elastically and transparently scale to hundreds of nodes for critical workloads without changing business logic. And you can scale your performance and storage nodes separately according to your business needs.
```

# TiDB Cloud Introduction

> [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) is a fully-managed Database-as-a-Service (DBaaS) that brings [TiDB](https://docs.pingcap.com/tidb/stable/overview), an open-source Hybrid Transactional and Analytical Processing (HTAP) database, to your cloud. TiDB Cloud offers an easy way to deploy and manage databases to let you focus on your applications, not the complexities of the databases. You can create TiDB Cloud clusters to quickly build mission-critical applications on Google Cloud Platform (GCP) and Amazon Web Services (AWS).

Reviewer 2: ```suggestion
[TiDB Cloud](https://www.pingcap.com/tidb-cloud/) is a fully-managed Database-as-a-Service (DBaaS) that brings [TiDB](https://docs.pingcap.com/tidb/stable/overview), an open-source Hybrid Transactional and Analytical Processing (HTAP) database, to your cloud. TiDB Cloud offers an easy way to deploy and manage databases to let you focus on your applications, not the complexities of the databases. You can create TiDB Cloud clusters to quickly build mission-critical applications on Google Cloud Platform (GCP) and Amazon Web Services (AWS).
```

![TiDB Cloud Overview](/media/tidb-cloud/tidb-cloud-overview.png)

## Why TiDB Cloud

TiDB Cloud allows you with little or no training to handle complex tasks such as infrastructure management and cluster deployment easily.

> - Developers and database administrators (DBAs) can rapidly analyze a large volume of data across multiple datasets.

Reviewer 3: Developers and database administrators (DBAs) can handle large amount of online traffic effortlessly as well as rapidly analyze a large volume of data across multiple datasets.

> - Developers and database administrators (DBAs) can rapidly analyze a large volume of data across multiple datasets.

Author: ```suggestion
- Developers and database administrators (DBAs) can handle a large amount of online traffic effortlessly and rapidly analyze a large volume of data across multiple datasets.
```

# TiDB Cloud Introduction

> [TiDB Cloud](https://en.pingcap.com/tidb-cloud/) is a fully-managed Database-as-a-Service (DBaaS) that brings [TiDB](https://docs.pingcap.com/tidb/stable/overview), an open source Hybrid Transactional and Analytical Processing (HTAP) database, to your cloud, and lets you focus on your applications, not the complexities of your database. You can run TiDB clusters on Google Cloud Platform (GCP) and Amazon Web Services (AWS).

Reviewer 2: ```suggestion
[TiDB Cloud](https://en.pingcap.com/tidb-cloud/) is a fully-managed Database-as-a-Service (DBaaS) that brings [TiDB](https://docs.pingcap.com/tidb/stable/overview), an open-source Hybrid Transactional and Analytical Processing (HTAP) database, to your cloud, and lets you focus on your applications, not the complexities of your database. You can run TiDB clusters on Google Cloud Platform (GCP) and Amazon Web Services (AWS).
```
